categories/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = 'categories'
+
+class Sub_Category(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = 'sub_categories'
```

Comment:
subcategory의 경우 하나의 단어로 사용할 수 있을 것 같아요~
```suggestion
        db_table = 'subcategories'
```